include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -206,6 +344,17 @@ class SplineEvaluator
                 });
     }
 
+    /** @brief Perform batched 1D integrations of a spline function (described by its spline coefficients) along the dimension of interest and store results on a subdomain of batch_domain.
+     *
+     * The spline coefficients represent a spline function in a B-splines (basis splines). They can be obtained using a SplineBuilder.
+     *
+     * The integration is not performed in a multidimensional way (in any sense). This is a batched 1D integration. It means for each element of integrals, the integration is performed with the 1D set of spline coefficents of spline_coef identified with the same DiscreteElement.
+     *
+     * @param[out] integrals The integrals of the spline function on the subdomain of batch_domain. For practical reasons those are
+     * stored in a Chunkspan defined on a batch_domain_type. Note that the coordinates of the
+     * points represented by this domains are unused and irrelevant.
+     * @param[in] spline_coef A Chunkspan storing the spline coefficients.
```

Comment:
```suggestion
     * stored in a ChunkSpan defined on a batch_domain_type. Note that the coordinates of the
     * points represented by this domains are unused and irrelevant.
     * @param[in] spline_coef A ChunkSpan storing the spline coefficients.
```

`ChunkSpan` not `Chunkspan` as this is a code keyword